vault.py

```diff
@@ -74,22 +74,30 @@ def run(self, terms, inject=None, variables=None, **kwargs):
             raise AnsibleError('Vault authentication token missing. Specify with'
                                ' VAULT_TOKEN environment variable or $HOME/.vault-token')
 
+        cafile = os.getenv('VAULT_CACERT')
+        capath = os.getenv('VAULT_CAPATH')
         try:
-            context = ssl.create_default_context(cafile=os.getenv('VAULT_CACERT'),
-                                                 capath=os.getenv('VAULT_CAPATH'))
+            if cafile or capath:
+                context = ssl.create_default_context(cafile=cafile, capath=capath)
+            else:
+                context = None
             request_url = urljoin(url, "v1/%s" % (key))
             req = urllib2.Request(request_url, data)
             req.add_header('X-Vault-Token', token)
             req.add_header('Content-Type', 'application/json')
-            response = urllib2.urlopen(req, context=context)
+            if context:
```

Comment:
Remove if statement and do `response = urllib2.urlopen(req, context=context)`. It should work with a None context.